ansible_ai_connect_chatbot/package.json

```diff
@@ -3,10 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@microsoft/fetch-event-source": "^2.0.1",
     "@patternfly/chatbot": "^2.2.0-prerelease.17",
-    "@testing-library/jest-dom": "^5.17.0",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
```

Comment:
Removed jest-dom, which is not used, and moved other two test libraries from `dependencies` to `devDependencies`.